pkgs/tools/networking/doggo/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  runCommand,
+  installShellFiles,
+}:
+buildGoModule rec {
```

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
, runCommand
, installShellFiles
}:

buildGoModule rec {
```

nixpkgs does not support the contradicting formatting style of alejandra.